src/main/java/Duke.java

```diff
@@ -1,10 +1,209 @@
+
+                if (done == "1") {
+                    isDone = true;
+                }
+
+
+                if (function == "D") {
+                    String description = sc.next();
+                    String date = sc.next();
+                    loadingTask = new Deadline (description, date, isDone);
+                    taskings.add(loadingTask);
+                } else if (function == "T") {
+                      String restOfDescription = sc.next();
+                    loadingTask = new Todo (restOfDescription,isDone);
+                    taskings.add(loadingTask);
+                } else if (function == "E") {
+                    String description = sc.next();
+                    String date = sc.next();
+                    loadingTask = new Event (description, date, isDone);
+                    taskings.add(loadingTask);
+                }
+        }
+
+            sc.close();
+
+        } catch (FileNotFoundException e) {
+            e.printStackTrace();
+        }
+
+
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Duke\n");
+        System.out.println("What can I do for you?\n");
+        Scanner in = new Scanner(System.in);
+
+        //user inputs into line string
+        String line;
+
+
+
+        Task[] list;
+        list = new Task[100];
+
+        //variables for list
+        int itemNumber = 0;
+        int itemDone = 0;
+        int numeration = 1;
+        int taskNumber = 0;
+
+        //variables to check command
+        boolean startsWithBye = false;
+        boolean startsWithList = false;
+        boolean startsWithDone = false;
+        boolean startsWithTodo = false;
+        boolean startsWithDeadline = false;
+        boolean startsWithEvent = false;
+        boolean startsWithDelete = false;
+        boolean isDone = false;
+
+        //other variables
+        int indexOfBackslash = 0;
+        String deadlineDate;
+
+
+        while (!startsWithBye) {
+
+            //waiting for user input
+            line = in.nextLine();
+
+            //checking command
+            startsWithBye = line.startsWith("bye");
+            startsWithList = line.startsWith("list");
+            startsWithDone = line.startsWith("done");
+            startsWithDeadline = line.startsWith("deadline");
+            startsWithTodo = line.startsWith("todo");
+            startsWithEvent = line.startsWith("event");
+            startsWithDelete = line.startsWith("delete");
+
+            try {
+
+                if (startsWithList) { //list command given
+
+                    System.out.println("Here are the tasks in your list:");
+                    for (numeration = 0; numeration < itemNumber; numeration++) {
```

Comment:
Maybe consider renaming the variable `numeration` to provide more details about what the data is about to help improve readability